IFC4x3/Sections/Domain specific data schemas/Schemas/IfcElectricalDomain/PropertySets/Pset_CableCarrierSegmentTypeConduitSegment/DocPropertySet.xml

```diff
@@ -11,6 +11,9 @@
 		<DocProperty xsi:nil="true" href="NominalHeight_3VRBe0qSOHuO00025QrEV" />
 		<DocProperty xsi:nil="true" href="ConduitShapeType_3b_p60qSOHuO00025QrEV" />
 		<DocProperty xsi:nil="true" href="IsRigid_3qIKs0qSOHuO00025QrEV" />
+		<DocProperty xsi:nil="true" href="NominalDiameter_2EWhC0qX8HuO00025QrEV" />
+		<DocProperty xsi:nil="true" href="InnerDiameter_2WYzjllJPDFg5iywvnD5NL" />
+		<DocProperty xsi:nil="true" href="OuterDiameter_2BiLROIs1Bl8QxhG83MXVt" />
```

Comment:
Inner and outer are most definitely quantities at a minimum I know it is easier to put them here, but we need to maintain purpose of the pset, so make a Qto_ConduitSegmentBaseQuantities